tempesta_fw/http.c

```diff
@@ -3475,6 +3491,7 @@ tfw_h2_adjust_req(TfwHttpReq *req)
 	}
 
 	r |= tfw_msg_write(&it, &h_via);
+	r |= tfw_msg_write(&it, &h_cl);
```

Comment:
Unconditional additions actually breaks rules for message framing in http1:

> A user agent SHOULD NOT send a
   Content-Length header field when the request message does not contain
   a payload body and the method semantics do not anticipate such a
   body.

> If a message is received that has multiple Content-Length header
   fields...
   ... recipient MUST either reject the message as invalid or replace the
   duplicated field-values 

Also Content-Length mustn't appear if chunked transfer encoding is used. This is not our use case since chunked encoding is prohibited for H2, but we actually never check that.

To fix that:
- add appropriate condition here
- add check for chunked encoding during request parsing, and reject request or connection if chunked encoding is used.